airflow/datasets/manager.py

```diff
@@ -226,14 +226,14 @@ def _send_dag_priority_parsing_request_if_needed(fileloc: str) -> str | None:
                 return None
             return req.fileloc
 
-        (_send_dag_priority_parsing_request_if_needed(fileloc) for fileloc in file_locs)
+        [_send_dag_priority_parsing_request_if_needed(fileloc) for fileloc in file_locs]
```

Comment:
```suggestion
        for fileloc in file_locs:
            _send_dag_priority_parsing_request_if_needed(fileloc)
```

Easier code, less risk to mistakes.